packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -572,7 +587,7 @@ class TextStyle with Diagnosticable {
   /// prefixed with 'packages/package_name/' (e.g. 'packages/cool_fonts/Roboto').
   /// The package name should be provided by the `package` argument in the
   /// constructor.
-  List<String>? get fontFamilyFallback => _package != null && _fontFamilyFallback != null ? _fontFamilyFallback!.map((String str) => 'packages/$_package/$str').toList() : _fontFamilyFallback;
+  List<String>? get fontFamilyFallback => _package != null ? _fontFamilyFallback?.map((String str) => 'packages/$_package/$str').toList() : _fontFamilyFallback;
```

Comment:
This might be slightly easier to ready if the simple case was first:
```dart
List<String>? get fontFamilyFallback => _package == null ? _fontFamilyFallback :  _fontFamilyFallback?.map((String str) => 'packages/$_package/$str').toList();
```